README.md

```diff
@@ -1 +1,25 @@
-# docker-guestfs
\ No newline at end of file
+# docker-guestfs
```

Comment:
Can you add some notes about goal/purpose of this repository the corresponding `Dockerfile` and the resulting container image. 

Otherwise it will be hard in a few months to understand the intention of this repo (imo).

Reply:
I will add it, but it's simply a docker image with libguestfs installed.

Reply:
Added.